packages/atomic/src/components/common/interface/store.ts

```diff
@@ -136,7 +136,8 @@ export function createAtomicCommonStore<
     getFacetElements() {
       return stencilStore
         .get('facetElements')
-        .filter((element) => isInDocument(element));
+        .filter((element) => isInDocument(element))
+        .filter((element) => isVisible(element));
```

Comment:
Does this copy dependant facets?

E.g., if you have facet A and facet B, and facet B only shows up when facet A has a selected value, are you able to:
1. Open the refine modal
2. Select a value in facet A
3. See facet B

Reply:
Good catch, no it doesnt copy dependant facets. I will try to adjust my solution